docs/graphql-api/queries/aggregation-queries.mdx

```diff
@@ -0,0 +1,114 @@
+---
+sidebar_label: Aggregation queries
+sidebar_position: 3
+description: Make aggregation queries with Hasura DDN.
+keywords:
+  - hasura
+  - docs
+  - postgres
+  - query
+  - aggregation query
+---
+
+import GraphiQLIDE from "@site/src/components/GraphiQLIDE";
+
+# Aggregation Queries
+
+## **Aggregate** fields
+
+You can fetch aggregations on columns along with nodes using an aggregation query.
+
+The **name of the aggregate field** is of the form `<field-name>Aggregate`.
+
+Common aggregation functions are `count`, `sum`, `avg`, `max`, `min`, etc. You can see the complete specification of the
```

Comment:
Can we do these? Is it up to the connector? At the moment only count is coming through on postgres.

Reply:
`_count` is auto-generated for PG, but I believe that as a user, you can define whatever you'd like. I'll double-check before deploying 👍

Reply:
You may only be seeing `_count` because of the type. Here's an auto-generated valid example for a number field:

```graphql
query SchrutebucksAverage {
  schrutebucks_aggregate {
    quantity {
      avg
    }
  }
}
```

That returns:

```json
{
  "data": {
    "schrutebucks_aggregate": {
      "quantity": {
        "avg": 6.625
      }
    }
  }
}
```

With these fields available:

```graphql
query SchrutebucksAverage {
  schrutebucks_aggregate {
    quantity {
      avg
      _count
      _count_distinct
      bit_and
      bit_or
      bit_xor
      max
      min
      stddev
      stddev_pop
      stddev_samp
      sum
      var_pop
      var_samp
      variance
    }
  }
}
```